saleor/graphql/account/mutations/base.py

```diff
@@ -414,6 +359,43 @@ def post_save_action(cls, info: ResolveInfo, instance, cleaned_input):
             if user_gift_cards := get_user_gift_cards(instance):
                 mark_gift_cards_search_index_as_dirty(user_gift_cards)
 
+    @classmethod
+    def process_account_confirmation(
```

Comment:
I would explicitly use name `set_password`. It is the exact action that this method is doing. `account_confirmation` can be misleading, as we already have an event like: `send_account_confirmation`. 
Additionally as this is the event triggered by staff (at least it is what I see in the code), maybe worth to add `customer` instead of `account`. `account` suggest me that I am operating on my own account (as customer's mutation to manage the account, starts with prefix `account`)